packages/chaire-lib-common/src/utils/osm/OsmOverpassDownloader.ts

```diff
@@ -119,6 +141,38 @@ class OsmOverpassDownloaderImpl implements OsmOverpassDownloader {
         return osmToGeojson(jsonContent) as GeoJSON.FeatureCollection;
     }
 
+    /**
+     * Download data from openstreetmap API, convert it to geojson and write it to
+     * a geojson file
+     * If the file exist, it will be overwritten
+     *
+     * See downloadGeojson for params
+     */
+    public async writeGeojson(
```

Comment:
For more clarity, maybe call the methods that download and write date downloadAndWrite<Format>? or fetchAndWrite

Reply:
That would imply also renaming all the one that existed before right?

Reply:
In a world where method names should be self-explaining, then I guess so.